src/components/Collections/CollectionShare/publicShare/manage/deviceLimit.tsx

```diff
@@ -24,24 +25,58 @@ export function ManageDeviceLimit({
             deviceLimit: newLimit,
         });
     };
-
+    const [shareDeviceLimitModalView, setDeviceLimitModalView] =
+        useState(false);
+    const [shareDeviceLimitValue, setDeviceLimitValue] = useState(
```

Comment:
we can directly use `publicShareProp.deviceLimit` no, why create a new state here?